.github/workflows/query-engine-driver-adapters.yml

```diff
@@ -31,8 +31,6 @@ jobs:
             setup_task: 'dev-neon-ws-postgres13'
           - name: 'libsql'
             setup_task: 'dev-libsql-sqlite'
-          - name: 'planetscale'
-            setup_task: 'dev-planetscale-vitess8'
```

Comment:
Intentionally dropping something that is currently in `main`?

Reply:
Yes, this doesn't provide any value at all until fituring out the vitess test situation for driver adapters. Which has it's own issue. As of now, the feedback on this test suite doesn't provide any kind of value.

BTW: PS tests pass on prisma/prisma, but I expect a lot of tests broken for planetscale in the engines.